### PR TITLE
#170529958 add user profile picture 

### PR DESCRIPTION
**Pull request by higustave-ops**
#### What does this PR do?
`add user profile picture`
#### Description of Task to be completed?
`To create a user account profile page settings feature by allowing a user to be able to edit his/her profile picture`
#### How should this be manually tested?
* Clone this repo
* Navigate in the repo root using `CMD` or any `TERMINAL`
* Install all packages using `npm install`
* Type `npm run dev` for Postman testing And `npm run test` for mocha testing
* Open postman choose PATCH `localhost:3000/api/edit-profile` to edit profile
* Click `body` then `x-www-form-urlencoded`
* Fill in all values you want to edit from the profile for all or some of the following properties: `profilePicture`
* Click `SEND` on the top right corner of Postman

#### What are the relevant pivotal tracker stories?
- [#170529958](https://www.pivotaltracker.com/story/show/170529958)

#### Screenshots (if appropriate)
`Postman screenshot on Edit Profile endpoint`
Screen Shot 2020-01-06 at 11 30 43

### CODE REVIEW

File: src/controllers/profile-settings.controller.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import ResponseService from '../services/response.service';
 import UserService from '../services/user.service';
 import JwtService from '../services/jwt.service';
+
```
Comment (victor-abz) [inline src/controllers/profile-settings.controller.js]: Please get rid of unnecessary changes.

**Review by victor-abz — APPROVED** [review]
LGTM

**Review by erickyvand — APPROVED** [review]

**Comment by erickyvand** [comment]
LGTM

**Comment by adesege** [comment]
The changes here are already included in #33. Is there a reason for duplicating the changes?

**Comment by higustave-ops** [comment]
> The changes here are already included in #33. Is there a reason for duplicating the changes?

Sorry for duplication. I was working on #33 and by mistake checked out to #34

**Comment by adesege** [comment]
I will merge this PR but you should remove the changes in the other PR.
